### PR TITLE
[de] add str / strasse to LineExpander.java

### PR DESCRIPTION
**Pull request by AgnesKleinhans**
Related PR: [[de] add street names](https://github.com/languagetool-org/languagetool/commit/1bf2271d3a2cfbb020a3c6bedfbb39c301f52338#commitcomment-121961848)

@danielnaber Could you please review this PR? :-) Thank you!

### CODE REVIEW

File: languagetool-language-modules/de/src/main/java/org/languagetool/rules/de/LineExpander.java [hunk 1]
```diff
@@ -122,6 +122,11 @@ private List<String> handleLineWithFlags(String line) {
       } else if (c == 'F') {
         add(result, word);
         result.add(word + "in"); // (m/f)
+       } else if (c == 'T') {
+        add(result, word);
```
Comment (danielnaber) [inline languagetool-language-modules/de/src/main/java/org/languagetool/rules/de/LineExpander.java:126]: there should be 2-space indentation here inside the `if`

File: languagetool-language-modules/de/src/main/java/org/languagetool/rules/de/LineExpander.java [hunk 2]
```diff
@@ -122,6 +122,11 @@ private List<String> handleLineWithFlags(String line) {
...
+        if (word.endsWith("straße") || word.endsWith("strasse"))  {
+        result.add(word.replaceAll("stra(ß|ss)e", "str."));
```
Comment (danielnaber) [inline languagetool-language-modules/de/src/main/java/org/languagetool/rules/de/LineExpander.java:128]: 2-space indentation

**Review by danielnaber — CHANGES_REQUESTED** [review]
Looks good already, but could you also add a test to `LineExpanderTest` for this new flag?

**Comment by danielnaber** [comment]
Also maybe clean up the `spelling.txt` as it has `str.` <-> `straße` duplicates now that should use the `/T` now instead.

**Comment by AgnesKleinhans** [comment]
Thank you!! :-) I added a test to `LineExpanderTest`; I will clean up `spelling.txt` in a separate commit!

**Pull request merged**